README.md

```diff
@@ -1,62 +1,54 @@
 
 - [Overview](#overview)
-  - [Demonstrated Features](#demonstrated-features)
+- [Demonstrated Features](#demonstrated-features)
 - [Getting Started](#getting-started)
   - [Prerequisites](#prerequisites)
-  - [Importing the Project in Eclipse](#importing-the-project-in-eclipse)
-  - [Building and Running](#building-and-running)
+  - [Clone, Build & Run](#clone-build--run)
+  - [Using the Application](#using-the-application)
+  - [Using Eclipse](#using-eclipse)
   - [Database Setup and Spring Profiles](#database-setup-and-spring-profiles)
   - [Using a File-Based SQLite Database](#using-a-file-based-sqlite-database)
 - [Get Support](#get-support)
 - [License](#license)
 
 # Overview
 
-This sample application shows how to conveniently create business applications based on **CDS domain models**, persisting data with **SQLite**, or **SAP HANA**, and exposing an **OData V4** frontend with an **SAP Fiori** frontend on top.
-
-This sample uses Spring Boot as an **application framework**. Although a CAP Java application isn’t required to build on Spring Boot, it’s the first choice of framework, as it’s seamlessly integrated.
+This sample uses **Spring Boot** as an application framework. Although CAP Java applications aren't required to build on Spring Boot, it’s the first choice of framework, as it’s seamlessly integrated.
 
-The **domain models** are defined using [CDS entity definitions](https://cap.cloud.sap/docs/cds/cdl#entity-and-type-definitions).
+The **domain models** are defined using **[CDS entity definitions](https://cap.cloud.sap/docs/cds/cdl#entity-and-type-definitions)**. By default, data is stored in an in-memory or optionally a file-based **SQLite** database. Once productively deployed to the SAP Cloud Platform, you can easily switch to **SAP HANA**.
 
-By default, an in-memory or optionally a file-based SQLite database is used for **data persistency**. Once productively deployed to SAP Cloud Platform, SAP HANA can be used.
+The data is exposed using **[CDS Service Models](https://cap.cloud.sap/docs/cds/cdl#services)**. The **OData V4 protocol adapter** translates the CDS service models into corresponding OData schemas and maps the incoming OData requests to the corresponding CDS services.
 
-**Services** are defined using [CDS Service Models](https://cap.cloud.sap/docs/cds/cdl#services). The **OData V4 Protocol Adapter** translates the CDS service models into corresponding OData schemas and maps the incoming OData requests to the corresponding CDS services.
+Custom **busines logic** is implemented in by so called **[custom event handlers](https://cap.cloud.sap/docs/get-started/in-a-nutshell#adding-custom-logic)**, while most **CRUD requests** are served out-of-the-box.
```

Comment:
Custom **busines logic** is implemented in so called **[custom event handlers](https://cap.cloud.sap/docs/get-started/in-a-nutshell#adding-custom-logic)**

removed **by**